samples/batch.js

```diff
@@ -40,21 +45,17 @@ function createQueryPartitions(instanceId, databaseId, identifier, projectId) {
 
   const query = 'SELECT * FROM Singers';
 
-  transaction
-    .createQueryPartitions(query)
-    .then(data => {
-      const partitions = data[0];
-      console.log(
-        `Successfully created ${partitions.length} query partitions.`
-      );
-    })
-    .catch(err => {
-      console.error('ERROR:', err);
-    });
+  try {
+    const data = await transaction.createQueryPartitions(query);
+    const partitions = data[0];
+    console.log(`Successfully created ${partitions.length} query partitions.`);
+  } catch (err) {
```

Comment:
Let's just drop the `try/catch` stuff unless there's a finally we really need to execute.